hack/utils/dupissue/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+.DEFAULT_GOAL:=help
+
+help: ## Display this help message
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[0-9A-Za-z_-]+:.*?##/ { printf "  \033[36m%-45s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+lint: ## Run Go code linting. Requires the $GOLANGCI_LINT env var to be set.
+ifeq ($(origin GOLANGCI_LINT),undefined)
+	@echo "Error! GOLANGCI_LINT env var not set"
+else
+	$(GOLANGCI_LINT) run -v --timeout=5m
+endif
+
+get-deps: ## Get all go dependencies
+	go mod tidy
+
+test:
+	@echo "N/A: No unit tests for hack/packages"
+
+e2e-test:
+	@echo "N/A: No e2e tests for hack/packages"
+
+build: ## Build the executable
+	go build -o dupissue dupissue.go
+
+run: ## Run the dupissue program to duplicate issues to the tanzu framework repo
+	@echo "N/A: No e2e tests for hack/packages"
```

Comment:
Should this chunk get updated to print that the user should build the binary first (to then be able to pass arguments to it)?

Reply:
changed the comment around so there isnt a run command. the idea is the user builds the utility and drops it somewhere on their filesystem. we dont want to run from the makefile here

Reply:
+1